workspaces/cli-config/tests/conflict-specs/ReadSpecification.test.ts

```diff
@@ -2,10 +2,17 @@ import * as path from 'path';
 import {resolveConflictsIfPresent} from "../../src/helpers/read-specification-json";
 // @ts-ignore
 import fs from "fs-extra";
+import {getSpecEventsFrom} from "../../build/helpers/read-specification-json";
```

Comment:
seems like this conflicts with the first import, it should be one or the other; presumably src

Reply:
REsolved.